### DISCUSSION TITLE
Loki/Prometheus query builder: For binary operations run partial queries to figure out labels/joins options

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#52097] [Observability]
For Loki/Prometheus binary operations in query builder - run partial queries to figure out labels/joins options.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
